mock-core-configs/etc/mock/templates/centos-stream-8.tpl

```diff
@@ -4,7 +4,7 @@ config_opts['releasever'] = '8'
 config_opts['package_manager'] = 'dnf'
 config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
 config_opts['bootstrap_image'] = 'registry.centos.org/centos:8'
-config_opts['dnf_vars'] = { 'stream': '8-stream',
+config_opts['dnf_vars'] = { 'stream': 'stream-8',
```

Comment:
This variable should stay `8-stream`, as it's used by the mirrorlist service.

Reply:
yes, that's what the Stream came with and as mirrorlist will pick path appearing on mirror network, let's stick with 8-stream (as that's also how it's done on Stream itself for mirrorlist .. let's not confuse more :)